tests/app/submitter/test_convert_payload_0_0_3.py

```diff
@@ -390,7 +390,7 @@ def test_primary_person_list_item_conversion(fake_questionnaire_store):
 
     output = convert_answers(schema, fake_questionnaire_store, routing_path)
 
-    data_dict = json.loads(json.dumps(output['data'], for_json=True))
+    data_dict = json.loads(json.dumps(output['data']['answers'], for_json=True))
```

Comment:
Now that the list items are in the payload we should add tests for them as well as answers.